ucm2/USB-Audio/Behringer/UMC202HD-HiFi.conf

```diff
@@ -0,0 +1,116 @@
+Include.pcm_split.File "/common/pcm/split.conf"
+
+Macro [
+	{
+		SplitPCM {
```

Comment:
This is not required. There is no requirement to split the stream which has already 2 channels.